README.md

```diff
@@ -278,6 +278,25 @@ Windows Registry Editor Version 5.00
 "CertFile"="C:\Absolute\Path\To\SslCertificate"
 ```
 
+_**NOTE: It is important from a security perspective to ensure that
```

Comment:
Not a blocker here, but I'm wondering if we have to ensure that server-provided info is stored in the registry by us with these permission levels.

Reply:
Yeah, that would be good to check. Would it be alright to log this as a followup issue?